server/schedule/placement/rule.go

```diff
@@ -65,8 +65,9 @@ type Rule struct {
 	LabelConstraints []LabelConstraint `json:"label_constraints,omitempty"` // used to select stores to place peers
 	LocationLabels   []string          `json:"location_labels,omitempty"`   // used to make peers isolated physically
 	IsolationLevel   string            `json:"isolation_level,omitempty"`   // used to isolate replicas explicitly and forcibly
-
-	group *RuleGroup // only set at runtime, no need to {,un}marshal or persist.
+	Version          uint64            `json:"version,omitempty"`           // only set at runtime, add 1 each time rules updated, begin from 0.
+	CreateTimestamp  uint64            `json:"create_timestamp,omitempty"`  // only set at runtime, recorded rule create timestamp
```

Comment:
What does it used for?

Reply:
we use `createTimestamp` and `version` to compare whether rule is updated. This will avoid thinking rule unchanged after it created,deleted,re-created.

Reply:
Seem `createTimestamp` is not used to do the comparison?

Reply:
we have not introduced comparison yet.